swift/ql/src/queries/Security/CWE-079/UnsafeWebViewFetch.qhelp

```diff
@@ -3,6 +3,7 @@
   "qhelp.dtd">
 <qhelp>
 <overview>
+
```

Comment:
On line 29, you need a full stop after `</a>`

Reply:
Well spotted.  Fixed.